WorkersRights_v/Input/Holidays/UkrainianHolidays.txt

```diff
@@ -298,7 +298,7 @@ new year's day
 2013
 01
 1 
-end 
+end
```

Comment:
Perhaps you should consider changing the parsing code so as to "chomp" extra spaces at the beginnings and ends of lines to avoid this kind of easy-to-introduce bug?

Reply:
Hi Eyal,
I already changed the parser to check:
if (sLine.Left(3) == "end")

Reply:
1. Wouldn't that just check the last three characters, spaces included? I don't know 
2. A similar problem (though a bit less likely) could occur with the beginning of the line.